addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -3633,7 +3633,21 @@ options.registry.sizing.include({
             $body.on('mouseup', documentMouseUp);
         });
         return defs;
-    }
```

Comment:
Commit message: 
> The generated entries

  -> You should specify that it is the Navbar and the Content (like in the previous commit message) to be less confusing, since the paragraphs can still be drag and dropped :wink: You should also explain why we don't want this behavior :) 
  
  > the drag icon
  
  -> I would say "handle" rather than "icon"
  
  > , and only keeping the move arrows.
  
  -> Nitpicking but I would say "Note that they can still be moved using the arrows."